### ISSUE TITLE
打开enable_cluster_plan，insert into select方式插入数据，报不支持roundrobin

### ISSUE THREAD

**Issue by mass3** [ADBSQL/AntDB#36]
### 打开enable_cluster_plan，insert into select方式插入数据，报不支持roundrobin
```
postgres=# set enable_cluster_plan = on;
SET
postgres=# INSERT INTO fast_emp4000 SELECT * FROM slow_emp4000;
ERROR:  not support robin yet!
postgres=# set enable_cluster_plan = off;
SET
postgres=# INSERT INTO fast_emp4000 SELECT * FROM slow_emp4000;
INSERT 0 3378
```
### 测试sql：
```
CREATE TABLE fast_emp4000 (
    home_base    box
);
CREATE INDEX grect2ind ON fast_emp4000 USING gist (home_base);

CREATE TABLE slow_emp4000 (
    home_base    box
);
COPY slow_emp4000 FROM '/home/mass2/pg9.6regress/data/rect.data';
INSERT INTO fast_emp4000 SELECT * FROM slow_emp4000;

```

**Comment by hsiaoby** [comment]
like #32, fixed by acb13f511adb0fffaa74608d6de7ddd788ad6303

**Comment by mass3** [comment]
打开enable_cluster_plan， 用insert into select方式向roundrobin表中插入数据，结果正确
```
postgres=# CREATE TABLE slow_emp4000 (
postgres(#     home_base    box
postgres(# );
CREATE TABLE
postgres=# COPY slow_emp4000 FROM '/home/mass2/pg9.6regress/data/rect.data';
COPY 3378
postgres=# INSERT INTO fast_emp4000 SELECT * FROM slow_emp4000;
INSERT 0 3378
postgres=# 

```
